example/lib/mobile_scanner_overlay.dart

```diff
@@ -1,228 +1,192 @@
+                              padding: const EdgeInsets.all(16.0),
+                              child: Align(
+                                alignment: Alignment.bottomCenter,
+                                child: Opacity(
+                                  opacity: 0.7,
+                                  child: Text(
+                                    overlayText,
+                                    style: const TextStyle(
+                                      backgroundColor: Colors.black26,
+                                      color: Colors.white,
+                                      fontWeight: FontWeight.bold,
+                                      fontSize: 24,
+                                      overflow: TextOverflow.ellipsis,
+                                    ),
+                                    maxLines: 1,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            controller: controller,
+                            scanWindow: scanWindow,
+                            errorBuilder: (context, error, child) {
+                              return ScannerErrorWidget(error: error);
+                            },
+                          ),
+                        ),
+                        CustomPaint(
+                          painter: ScannerOverlay(scanWindow),
+                        ),
+                        Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Align(
+                            alignment: Alignment.bottomCenter,
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                              children: [
+                                IconButton(
+                                  onPressed: () => controller.toggleTorch(),
+                                  icon: Icon(
+                                    Icons.flashlight_on,
+                                    color: controller.torchEnabled
+                                        ? Colors.yellow
+                                        : Colors.black,
+                                  ),
+                                ),
+                                IconButton(
+                                  onPressed: () => controller.switchCamera(),
+                                  icon: const Icon(
+                                    Icons.cameraswitch_rounded,
+                                    color: Colors.white,
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                        ),
+                      ],
+                    )
+                  : const Center(
+                      child: Text("Tap on Camera to activate QR Scanner")),
```

Comment:
There is a missing comma here, after the Text widget.

```suggestion
                      child: Text("Tap on Camera to activate QR Scanner"),),
```